src/Categories/Diagram/End/Properties.agda

```diff
@@ -22,6 +26,33 @@ private
     o ℓ e : Level
     C D E : Category o ℓ e
 
+module _ {o ℓ e o′ ℓ′ e′} {C : Category o ℓ e} {D : Category o′ ℓ′ e′}
+  (F : Bifunctor (Category.op C) C D) where
+  open Wedges F
+
+  -- Being an End is the same as being a Terminal object in the category of Wedges
```

Comment:
I think we can obtain most of the conclusions here via duality

Reply:
From the Coend case [which I readily believe], or from some other results?  I'm quite fine rewriting some of the proofs to go via duality, once we fix a convention of what results should be done 'by hand' and which should be via duality. We should be consistent which of the 'side' we (arbitrarily!) chose to do first.

Reply:
yeah it looks derivable from the coend case. I think the idea is just to save effort from dual proofs once we have one side already. I don't think one side is better than another, so this choice can be a bit arbitrary.